deployment/cdk/opensearch-service-migration/lib/service-stacks/reindex-from-snapshot-stack.ts

```diff
@@ -113,19 +124,54 @@ export class ReindexFromSnapshotStack extends MigrationServiceCore {
             servicePolicies.push(getSecretsPolicy);
         }
 
+        const volumes = [sharedLogFileSystem.asVolume()];
+        const mountPoints = [sharedLogFileSystem.asMountPoint()];
+
+        // Calculate the volume size based on the max shard size
+        // Have space for the snapshot and an unpacked copy, with buffer
+        const planningSize = props.maxShardSizeGiB ?? 80;
+        const volumeSize = Math.max(
+            Math.ceil(planningSize * 2 * 1.15),
+            1
+        )
+
+        if (volumeSize > 16000) {
+            // 16 TiB is the maximum volume size for GP3
+            throw new Error(`"Your max shard size of ${props.maxShardSizeGiB} GiB is too large to migrate."`)
+        }
+
+        // Volume we'll use to download and unpack the snapshot
+        const snapshotVolume = new ServiceManagedVolume(this, 'SnapshotVolume', {
+            name: 'snapshot-volume',
+            managedEBSVolume: {
+                size: Size.gibibytes(volumeSize),
+                volumeType: EbsDeviceVolumeType.GP3,
+                fileSystemType: FileSystemType.XFS,
+                tagSpecifications: [{
```

Comment:
EBS Console shows name which reflects the `Name` tag on the volume, can we populate it with something that says RFS and the stage

Reply:
it's case sensitive for `Name`
<img width="966" alt="image" src="https://github.com/user-attachments/assets/c4ad224a-9786-43f8-8f35-e08dd50c185f">
<img width="375" alt="image" src="https://github.com/user-attachments/assets/093e3302-8f3a-4384-99b1-524e11ab4673">